contrib/local/setup_wasmd.sh

```diff
@@ -22,5 +22,5 @@ for addr in "$@"; do
   wasmd add-genesis-account "$addr" "1000000000$STAKE,1000000000$FEE"
 done
 # submit a genesis validator tx
-(echo "$PASSWORD"; echo "$PASSWORD"; echo "$PASSWORD") | wasmd gentx validator --chain-id="$CHAIN_ID" --amount "250000000$STAKE"
+(echo "$PASSWORD"; echo "$PASSWORD"; echo "$PASSWORD") | wasmd gentx validator "250000000$STAKE" --chain-id="$CHAIN_ID"
```

Comment:
Oh, this is important part for the cosmjs tests.
I don't know how best to flag it, so @willclarktech @webmaster128 Note this when we test cosmjs against 0.14.0 wasmd

Reply:
Does this change anything for the caller of this script? We should get the change when bundled as part of the docker container.

Reply:
yes, this is part of the docker script as well.
Please note: due to the 🐛  in the sdk the workaround is to pass the staked tokens twice. As argument and as flag.